{{cookiecutter.repo_name}}/{{cookiecutter.readme}}

```diff
@@ -4,17 +4,22 @@
 [![PyPI Version](https://img.shields.io/pypi/v/{{ cookiecutter.distribution_name }})](https://pypi.org/project/{{ cookiecutter.distribution_name }}/)
 ![License](https://img.shields.io/pypi/l/{{ cookiecutter.distribution_name }}?color=blue)
 
-{{ cookiecutter.description }}
+{{ cookiecutter.description }}.
 
-Installation
-------------
+
+## Installation
```

Comment:
Regarding the Markdown heading style, I spend a **lot** of time looking at plain-text Markdown (i.e., not rendered), and I usually find the underlined headers _much_ more "scannable" and readable than `#` / `##`. To me, that readability is worth the prospect of potentially having to someday adjust the underline length, although in reality I don't think headings change often enough for that to be a significant concern. So… while not a huge deal either way, I'm not sure I totally dig this proposed change.

Reply:
Again, I've no issue reverted those proposed changes. I'll followed your preferences ! :)